packages/arb-token-bridge-ui/src/hooks/__tests__/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { RenderHookResult, act, renderHook } from '@testing-library/react'
+
+import { UseBalanceProps, useBalance } from '../useBalance'
+
+// this method is suggested by https://stackoverflow.com/a/73476929/5143717
+export const renderHookAsync = async ({
+  provider,
+  walletAddress,
+  wrapper
+}: UseBalanceProps & {
+  wrapper: React.ComponentType
+}) => {
+  let hook:
+    | RenderHookResult<ReturnType<typeof useBalance>, UseBalanceProps>
+    | undefined
+
+  await act(async () => {
+    hook = renderHook(
+      () =>
+        useBalance({
```

Comment:
We should either have `renderHookAsync` support generic hooks instead of hard-coding `useBalance`. Or this file and function should be specific to `useBalance` hook only and be colocated within `useBalance.test.tsx`